### PR TITLE
Simplify usage of credentials().issue() and ipex().grant()

### PR DESCRIPTION
**Pull request by lenkan**
I am working on multisig issuance using signify and found it very hard to follow the flow for multisig issuance => grant. To get a grasp of it I cleaned up the code for issue + grant and I think I found some complicated and repeated code.

In short:

- Adding IpexGrantArgs and IssueCredentialArgs/IssueCredentialResult interfaces.
- Simplifying the flow in credentials().issue() function to avoid unnecessary re-assigning of variables and merging of objects.
- Moving the attachment creation of the embeds into ipex().grant() function to make it easier to use from a caller. This was repeated in all scripts.
- Enabling the integration tests for multisig.test.ts and credential.test.ts
- Added the "Operation" type.

I think it should be all correct since it works. I also took inspiration from the commit here https://github.com/pfeairheller/signify-ts/commit/dc98330b7f31960f6b379997101f93353c88b3da to remove the redundant Saider arguments when creating the attachments.

If you do not like it, just let me know and I will close this. I don't necessarily need to get this merged. I just implemented it so that I could figure out how to do multisig credential issuance on my end and thought it was an improvement.

### CODE REVIEW

**Comment by rodolfomiranda** [comment]
This is a great improvement!. You help me cross out some tasks on my todo list. The change in the `ipex().grant()` simplify the life of implementors.

**Review by rodolfomiranda — APPROVED** [review]

**Comment by kentbull** [comment]
Yes, that's really good to know that you can recreate the Saider instances from the `.ked` attribute of an ACDC or an Anchor (anc).

**Comment by lenkan** [comment]
@rodolfomiranda @pfeairheller I have rebased this now on top of development. I rephrased the PR title to reflect the changes now that the integration test was already merged. But the PR description is still valid. Please have a look.

**Review by pfeairheller — APPROVED** [review]
